### PR TITLE
Move BlackOilOnePhaseIndices to Opm namespace

### PR DESCRIPTION
**Pull request by blattms**
to make it consistent with the rest. #530 development must have been started before the renaming in #532 and somehow the old namespace survived.

Downstream PR is coming.

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this opm-simulators=2118 please

**Comment by blattms** [comment]
Missed some changes downstream that just got force-pushed.

**Comment by blattms** [comment]
Somehow jenkins did not pick up the second build command. (Probably too smart). Will trigger a build downstream OPM/opm-simulators#2118

File: opm/models/blackoil/blackoilonephaseindices.hh [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 #include <cassert>
 
-namespace Ewoms {
+namespace Opm {
```
Comment (akva2) [inline opm/models/blackoil/blackoilonephaseindices.hh]: correct the closing comment.

**Comment by akva2** [comment]
apart from the minor this is good. as you say it came in after my namespace move, and i guess was implemented before we considered it.

File: opm/models/blackoil/blackoilonephaseindices.hh (hunk 1 above)
Comment (blattms) [reply opm/models/blackoil/blackoilonephaseindices.hh]: Should I then change the others, too?
```
$ find opm -type f | xargs grep Ewoms|grep namespace
opm/models/blackoil/blackoilonephaseindices.hh:namespace Ewoms {
opm/models/blackoil/blackoilonephaseindices.hh:} // namespace Ewoms
opm/simulators/linalg/combinedcriterion.hh:}} // end namespace Linear,Ewoms
opm/simulators/linalg/convergencecriterion.hh:}} // end namespace Linear, Ewoms
opm/simulators/linalg/fixpointcriterion.hh:}} // end namespace Linear, Ewoms
opm/simulators/linalg/istlpreconditionerwrappers.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/istlsolverwrappers.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/istlsparsematrixadapter.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/linearsolverreport.hh:}} // end namespace Linear, Ewoms
opm/simulators/linalg/parallelbasebackend.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/parallelbicgstabbackend.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/parallelbicgstabbackend.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/parallelistlbackend.hh:}} // namespace Linear, Ewoms
opm/simulators/linalg/residreductioncriterion.hh:}} // end namespace Linear,Ewoms
opm/simulators/linalg/weightedresidreductioncriterion.hh:}} // end namespace Linear,Ewoms
```

Comment (akva2) [reply opm/models/blackoil/blackoilonephaseindices.hh]: yes, if i missed those shame on me!

**Comment by blattms** [comment]
Fixed the end namespace comments, too.

**Comment by blattms** [comment]
backported to release via #578

**Pull request merged**